specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/preview/2020-11-01-preview3/phonenumbers.json

```diff
@@ -0,0 +1,1014 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Phone Number Administration Service",
+    "description": "LROs: Search, Purchase, Release, Change Capabilities",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2020-07-20-preview1"
+  },
+  "paths": {
+    "/availablePhoneNumbers/countries": {
+      "get": {
+        "tags": ["List available countries"],
+        "operationId": "PhoneNumberAdministration_GetCountries",
```

Comment:
"PhoneNumberAdministration" should be renamed to "PhoneNumbers". We should remove the word "Administration" from the client name